go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/cosmos/gogoproto v1.7.0
 	github.com/cosmos/ledger-cosmos-go v0.14.0
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.3.0
+	github.com/golang/mock v1.6.0
```

Comment:
[go.uber.org/mock](https://github.com/uber/mock) should be used over golang/mock as that is no longer maintained